docs/console/company-configuration/providers/extensions/orchestrator-generator.md

```diff
@@ -0,0 +1,245 @@
+---
+id: orchestrator-generator
+title: Orchestrator Generator
+sidebar_label: Orchestrator Generator
+---
+
+Capability: `Orchestrator Generator`
+
+An Orchestrator Generator can generate orchestrator's **files**, ready to **deploy**, which are saved in the **configuration repository**. For instance, it could generate Kubernetes files.
+If the Extension is enabled, the Console will send a **POST request** to its Provider's API Base URL to generate the deploy files. The request body will contain some information about the Project and the Configuration. The Console will expect to receive, in response, the files.
+
+## Request format
+
+### Headers
+
+For security concerns, the request is provided with a specific header, `X-Mia-Signature`, which can be used to validate that the payload comes from the Console.
```

Comment:
this has not been implemented, the api is done using the authentication method specified in the provider configuration. (bearer token or basic auth)

Reply:
removed, now I talk about authentication